EdvClient.js

```diff
@@ -1089,7 +1089,14 @@ export class EdvClient {
     ]);
     delete encrypted.content;
     delete encrypted.meta;
-    delete encrypted.stream;
+
+    if(stream) {
```

Comment:
I think this needs to be:

```suggestion
    if(encrypted.stream) {
```

Because `stream` is not set when we're doing the final document update (I don't think), after writing the chunks.

Reply:
Got it, changed.